### DISCUSSION TITLE
Kanboard v1.2.42

### DISCUSSION THREAD

**Discussion by kirill12322** [creecros/Mailmagik#52] [Q&A]
After, installing the plugin, setup and saving, there is no way to make changes. The settings menu looks like on the screenshot. What can be the reason?
![Снимок экрана 2024-11-21 190826](https://github.com/user-attachments/assets/00f3da46-1c99-43b1-aa97-cb216a16e004)

**Comment by alfredbuehler** [comment]
I guess there is some other plugin that breaks the CSS. I'd remove other plugins to narrow down the issue.

**Comment by kirill12322** [comment]
I uninstalled all plugins, install MAILMAGIK, setup in Settings > Email settings, click save, and have this problem again. After uninstalling MAILMAGIK, the Email settings page takes the old form. When other plugins are installed, after uninstalling MAILMAGIK, the Email settings page does not change its appearance and remains as it was when MAILMAGIK was installed.
![Снимок экрана 2024-11-22 010813](https://github.com/user-attachments/assets/c93126cc-e95a-4582-9fa8-60ae52e5c5a6)

**Comment by alfredbuehler** [comment]
OK, I see (and confirm) it. I'll take a more in-depth look.
